src/test/cypress/e2e/exercises/programming/ProgrammingExerciseParticipation.cy.ts

```diff
@@ -11,11 +11,15 @@ import {
 } from '../../../support/pageobjects/exercises/programming/OnlineEditorPage';
 import { convertCourseAfterMultiPart } from '../../../support/requests/CourseManagementRequests';
 
-// The user management object
+// Users
 const users = artemis.users;
+const admin = users.getAdmin();
+const studentOne = users.getStudentOne();
+const studentTwo = users.getStudentTwo();
+const studentThree = users.getStudentThree();
```

Comment:
(Maybe as follow up) Would it be possible to write a method getStudent(index: number) instead of all these methods?

Reply:
I think the idea here is to emphasis, that we only have three users. So if someone would try `getStudent(4)` it would fail later then we he tries to use the none existing `getStudentFour` method.